react-validated-input-component/my-app/src/Password.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { faCheck } from '@fortawesome/free-solid-svg-icons';
+import './Password.css';
+
+export default function Password() {
+  const [password, setPassword] = useState('');
+  const [valid, setValid] = useState(false);
+
+  function handleInput(e) {
+    const passwordVal = e.target.value;
+    setPassword(passwordVal);
+    if (passwordVal.length > 7) {
+      setValid(true);
+    } else if (valid) {
+      setValid(false);
+    }
+  }
```

Comment:
Since the current value of `valid` can be directly calculated based on the current value of `password`, avoid setting a `valid` state. That way, you reduce the chance of accidentally introducing inconsistent states later.